lib/ui/views/scanner_view.dart

```diff
@@ -47,4 +39,34 @@ class ScannerView extends StatelessWidget {
       ),
     );
   }
+
+  Widget buildProductList(
+      BuildContext context, int index, ScannerViewModel model) {
+    Product p = model.getScannedProducts()[index];
+    return InkWell(
```

Comment:
Można by dać jakiś padding żeby nie było jedno przy drugim i przy krawędziach np 8, tylko wtedy zmniejszy się ten obszar bo na stałe jest ustawione 100 - można pomyśleć czy nie da się bez ustawiania konkretnego rozmiaru

Reply:
Można opakować Image w AspectRatio to chociaż wtedy zaswsze będzie kwadratem i nie mamy na stałe wpisane 100